console/src/components/Main/utils.ts

```diff
@@ -1,3 +1,46 @@
+const INTERCOM_ID = 'rucirpb3';
+export const INTERCOM_URL = `https://widget.intercom.io/widget/${INTERCOM_ID}`;
+
+export const bootIntercom = () => {
+  if (!window.Intercom) return;
+  window.Intercom!('boot', {
+    app_id: INTERCOM_ID,
+  });
+};
+
+export const closeIntercom = () => {
+  if (window.Intercom) window.Intercom('shutdown');
+};
+
+export const startIntercom = () => {
+  if (window.Intercom) bootIntercom();
+
+  const head = document.getElementsByTagName('head')[0];
+  const script = document.createElement('script');
+  script.type = 'text/javascript';
+  script.onload = bootIntercom;
+  script.src = INTERCOM_URL;
+  head.appendChild(script);
+};
+
+const chatState = 'console:chatEnabled';
+
+export const persistChatState = (isEnabled: boolean) => {
+  window.localStorage.setItem(chatState, String(isEnabled));
```

Comment:
we have a utility for this `getLocalStorageItem`, we can re-use if you want, do as you wish :see_no_evil:

Reply:
I know, but... LS storage utils are all over the place and it needs some bigger cleanup. Will do it one day in another PR.